tests/unit/script/4-list-table-split.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -x
+set -e
+
```

Comment:
Let's add some comments about what we're trying to achieve (test the split decision and number of parts, and that table_3 is not to be split because we don't have an entry for it in our pgcopydb.table_size cache). Also please mention, as in the other scripts, that we're depending on the PGCOPYDB_SOURCE_PGURI environment variable in the script.